cos-kernel

```diff
@@ -400,30 +409,27 @@ subcmd_fetch() {
 
 
 subcmd_build() {
-	local path
-
 	subcmd_fetch
+	set_compilation_env
 
-	path="$(realpath "${TOOLCHAIN_DIR}/bin")"
-	${PRINT_CMD} export PATH="${path}:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/google-cloud-sdk/bin"
 	${PRINT_CMD} cd "${KERNEL_SRC_DIR}"
-	${PRINT_CMD} make ${MAKE_VERBOSE} -j $(($(nproc) * 2)) CROSS_COMPILE=x86_64-cros-linux-gnu-
```

Comment:
Per our offline conversation, we can't drop CROSS_COMPILE because we need it for the tools (nm, ar, size, etc.).  So we would set CC and CXX as well as CROSS_COMPILE.